### ISSUE TITLE
why the pre_model has no use?

### ISSUE THREAD

**Issue by bubulv** [maiminh1996/YOLOv3-tensorflow#10]
I convert my yolov3_final.weights for my own dataset to yolov3_final.h5, and I use python3 train.py for training.  But the loss is also very high. It's the same as I use the darknet53.weights.
My yolov3_final.weights is the final train weight for my dataset on darknet, in darknet my cfg and weights are ok and loss is very low.
If the yolov3_final.h5  converts right, the training loss should be low too.  The convert.py is download from https://github.com/qqwweee/keras-yolo3
Can you help me? Mybe I make some mistake?

**Comment by qiyayun** [comment]
@bubulv Hello ,I try to train my own data ,but loss is nan , Do you modify the code except data path during training?
